Libraries/LibWeb/HTML/HTMLScriptElement.cpp

```diff
@@ -639,6 +641,26 @@ void HTMLScriptElement::unmark_as_parser_inserted(Badge<DOM::Range>)
     m_parser_document = nullptr;
 }
 
+// https://www.w3.org/TR/trusted-types/#the-text-idl-attribute
+WebIDL::ExceptionOr<void> HTMLScriptElement::set_text(TrustedTypes::TrustedScriptOrString text)
+{
+    // 1. Let value be the result of calling Get Trusted Type compliant string with
+    //    TrustedScript, this’s relevant global object, the given value, HTMLScriptElement text, and script.
+    auto const value = TRY(get_trusted_type_compliant_string(
+        TrustedTypes::TrustedTypeName::TrustedScript,
+        HTML::relevant_global_object(*this),
+        text,
+        TrustedTypes::InjectionSink::HTMLScriptElementtext,
+        TrustedTypes::Script.to_string()));
+
+    // 2. Set this’s script text value to the given value.
+    m_string_text = value;
```

Comment:
I think calling the field `m_script_text` to match the spec term is a little less confusing.